app/controllers/maps_controller.rb

```diff
@@ -6,8 +6,6 @@ class MapsController < ApplicationController
 
   respond_to :html, :json, :csv
 
-  autocomplete :map, :name, full: true, extra_data: [:user_id]
```

Comment:
as far as I can tell, this creates a controller action autocomplete_map_name, which isn't used anywhere else in our app, and is part of the rails3-jquery-autocomplete. I could be wrong, but I think we have different actions and are using typeahead for all of our autocomplete now. 

I definitely put this up for review in case you know of cases why this is needed. Do we still need this line/gem?